<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Diagnostics</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="videowidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuResolution">
     <property name="title">
      <string>Resolution</string>
     </property>
    </widget>
    <addaction name="menuResolution"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Camera">
    <property name="title">
     <string>&amp;Camera</string>
    </property>
    <widget class="QMenu" name="menuSelect_Camera">
     <property name="title">
      <string>Select &amp;Camera</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuSelect_Frame_Rate">
     <property name="title">
      <string>Select &amp;Frame Rate</string>
     </property>
    </widget>
    <addaction name="menuSelect_Camera"/>
    <addaction name="menuSelect_Frame_Rate"/>
    <addaction name="actionCapture_Image"/>
    <addaction name="action_Reset"/>
    <addaction name="separator"/>
    <addaction name="action_Dump_Sensor_Registers"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Camera"/>
   <addaction name="menu_Window"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCapture_Image">
   <property name="text">
    <string>Capture &amp;Image</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action640x480">
   <property name="text">
    <string>640x480</string>
   </property>
  </action>
  <action name="action_Dump_Sensor_Registers">
   <property name="text">
    <string>&amp;Dump Sensor Registers</string>
   </property>
  </action>
  <action name="actionSelect_Camera">
   <property name="text">
    <string>Select Camera</string>
   </property>
  </action>
  <action name="actionG">
   <property name="text">
    <string>g</string>
   </property>
  </action>
  <action name="actionFf">
   <property name="text">
    <string>ff</string>
   </property>
  </action>
  <action name="actionS">
   <property name="text">
    <string>s</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
